akka-actor/src/main/scala/akka/actor/ActorCell.scala

```diff
@@ -661,11 +661,26 @@ private[akka] class ActorCell(
                or is missing an appropriate, reachable no-args constructor.
               """,
               i.getCause)
-          case x => throw ActorInitializationException(self, "exception during creation", x)
+          case x =>
+            val rootCause = rootCauseOf(x)
```

Comment:
I don't think there should be logic like this, the cause here is `x`, we should keep the full cause chain in case `x` was in turn caused by something else.